pandas/core/indexes/numeric.py

```diff
@@ -380,15 +380,6 @@ def get_loc(self, key, method=None, tolerance=None):
         if is_bool(key):
             # Catch this to avoid accidentally casting to 1.0
             raise KeyError(key)
-
-        if is_float(key) and np.isnan(key):
-            nan_idxs = self._nan_idxs
```

Comment:
IIRC this is the only non-test use of _nan_idxs, could remove

Reply:
Nice